### DISCUSSION TITLE
Is asciidoctor-web-pdf is available as maven plugin?

### DISCUSSION THREAD

**Discussion by jean-charles** [Mogztter/asciidoctor-web-pdf#688] [Q&A]
I chose to generate my curriculum-vitae using Maven as I do for my projects.

I'm currently using `org.asciidoctor.asciidoctor-maven-plugin`. But I'm not able to create a curriculum-vitae as the [one in the examples](https://github.com/ggrossetie/asciidoctor-web-pdf/tree/main/examples/resume), with a column on the left.

Is `asciidoctor-web-pdf` is available as maven plugin?

Regards

**Comment by ggrossetie** [comment]
Hello!

> Is asciidoctor-web-pdf is available as maven plugin?

I don't think so but it might be possible. You should probably ask at https://asciidoctor.zulipchat.com/
